content/documentation/staging/tutorial/05-connect/01-request-routing.adoc

```diff
@@ -0,0 +1,158 @@
+    <img src="/images/tutorial/05-01-route-to.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Add similar rules to route traffic from *viaggi.it* to *travels-v2* workload and from *voyages.fr* to *travels-v3* workload.
+
+When the three rules are defined you can use "Create" button to generate all Istio configuration needed for this scenario.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-rules-defined.png" title="Rules Defined">
+    <img src="/images/tutorial/05-01-rules-defined.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+The Istio Config linked with a given service can also be located under the "Istio" tab in the Service Details page.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-service-istio-config.png" title="Service Istio Config">
+    <img src="/images/tutorial/05-01-service-istio-config.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[4]* Verify Request Routing working from the *travels-portal* Graph
+
+Once the Request Routing is working we can verify that outbound traffic from every portal goes all to a single *travels* workload.
+
+If you use the "Find.../Hide..." filter, graph can hide old traffic grey edges for a clear graph.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-request-routing-graph.png" title="Travels Portal Namespace Graph">
+    <img src="/images/tutorial/05-01-request-routing-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Examining the "Inbound Traffic" for any of the *travels* workloads will show a similar pattern in the telemetry.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-travels-v1-inbound-traffic.png" title="Travels v1 Inbound Traffic">
+    <img src="/images/tutorial/05-01-travels-v1-inbound-traffic.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Using a large interval we can see how a workload received traffic from all portals and moved to a single portal once that the Request Routing scenarios was defined.
+
+NOTE: *[5]* Update or delete Istio Configuration
+
+Kiali Wizards allow to define high level Service Mesh scenarios and generate the Istio Configuration (VirtualServices, DestinationRules, Gateways and PeerRequests) needed for its implementation.
+
+These scenarios can be updated and the generated Istio configuration linked for a given service can be also deleted from the "Actions" menu.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-update-or-delete.png" title="Update or Delete">
+    <img src="/images/tutorial/05-01-update-or-delete.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+As part of the tutorial step, when you have finished to experiment more Routing Requests scenarios, you can delete the generated Istio config for the *travels* service.
```

Comment:
As part of this tutorial step, when you have finished experimenting with Routing Request scenarios then use the *travels* service "Actions" menu to delete the generated Istio config.